### DISCUSSION TITLE
Use 2.0 dev-server with eleventy 2.0.1

### DISCUSSION THREAD

**Discussion by ThePeterMick** [11ty/eleventy#3350] [Uncategorized]
Hi folks,

I noticed eleventy 2.0.1 makes use of dev-server 1.0.4 - is there a clean and simple way to bump it up to use 2.0 dev-server? I wanted to use the onRequest mappings and it seems only dev-server 2.0 provides that.

Thanks,
P

**Comment by zachleat** [comment]
Looking at the breaking changes for dev server 2.0 https://github.com/11ty/eleventy-dev-server/releases/tag/v2.0.0 I think it would probably work just to use npm’s overrides feature: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides
